src/flexbuffer_json.h

```diff
@@ -529,6 +529,7 @@ class JsonObject : JsonWithPath
         flexbuffers::TypedVector keys_ = flexbuffers::TypedVector::EmptyTypedVector();
         flexbuffers::Vector values_ = flexbuffers::Vector::EmptyVector();
         mutable tiny_bitset visited_fields_bitset_;
+        mutable std::unordered_map<std::string, JsonObject> child_objects_;
```

Comment:
This won't be accepted. This class is specially designed to be as cheap as possible to create/copy/move/delete. Adding a map type like this ruins that.

Reply:
Although I do appreciate the intent of trying to 'cache every object', I'm not actually sure it saves over the current model long term. It also gets complicated because as soon as someone makes a copy of an object they'll be deep copying the entire tree of objects instantiated at that point. Used field tracking in generic factory relies on the fresh instance having 'no fields visited' on the off chance the deserializer logic changes.

Reply:
I need to keep 4 JsonObjects on this class, how would you recommend I do that?

Reply:
Why exactly?